app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -585,7 +610,15 @@ class BrowserTabViewModel(
 
     private fun pageChanged(url: String, title: String?) {
         Timber.v("Page changed: $url")
+        val oldUrl = site?.url
```

Comment:
small rename if you want: oldUrl -> previousUrl

Reply:
Done!